### PR TITLE
Updating Footer

### PR DESCRIPTION
**Pull request by Shaba1406**
Done with the Changes.
-Updation in footer
-Text in centre
-Color Changes

issue no.- #51 

Screenshot:- 

Typing Test _ Test and practice your typing skills - Google Chrome 31-10-2022 15_43_26

### CODE REVIEW

**Comment by Shaba1406** [comment]
@arpitghura Done Please review it.

File: index.css [hunk 1]
```diff
@@ -132,18 +132,160 @@ li.nav-item{
...
+    background: #fd7014;
+  }
+  
+  .footer-socials .github:hover span a,
+  .footer-socials .github:hover .tooltip,
+  .footer-socials .github:hover .tooltip:before {
+    background: #fd7014;
+  }
+  
+  .footer-socials .email:hover span a,
```
Comment (arpitghura) [inline index.css:284]: Remove unused css

File: index.html [hunk 2]
```diff
@@ -7,6 +7,9 @@
     <title>Typing Test : Test and practice your typing skills</title>
     <link rel="stylesheet" href="index.css">
     <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet"href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"/>
+
```
Comment (arpitghura) [inline index.html:11]: Remove these extra lines

File: index.html [hunk 3]
```diff
@@ -56,9 +59,74 @@ <h3>Difficulty and Case Selection</h3>
...
+            <div class="icon email">
+              <div class="tooltip">Twitter</div>
+              <span
+                ><a href="https://twitter.com/arpitghura"
+                  ><i class="bi bi-twitter"></i></a
+              ></span>
+            </div>
+          </div>
+          <div class="text-center"style="color: #000000; padding-top: 30px; font-weight: 800">
+          <p style="font-size: 20px"center>Made with &hearts; by Arpit Ghura. All Rights Reserved &copy;</centere></p>
```
Comment (arpitghura) [inline index.html:126]: Decrease font size to 18px and use external css

**Review by arpitghura — CHANGES_REQUESTED** [review]
Make the same footer on all the pages and make sure the css should be written only once. 

You can create a new file named footer.css for that or also add footer css in index.js file. 

And do the requested changes.

**Comment by Shaba1406** [comment]
Done with the changes

**Review by arpitghura — CHANGES_REQUESTED** [review]
Screenshot_2022-10-31-23-01-56-21_40deb401b9ffe8e1df2f1cc5ba480b12
Breaking on Profile Page 


Screenshot_2022-10-31-23-02-07-79_40deb401b9ffe8e1df2f1cc5ba480b12
Improper alignment on other pages check for margin and padding.

**Comment by Shaba1406** [comment]
Sorry, I can't resolve this issue because I only have this much knowledge right now I am in a learning stage tried hard for it but it is breaking on other pages. :(

**Comment by arpitghura** [comment]
> Sorry, I can't resolve this issue because I only have this much knowledge right now I am in a learning stage tried hard for it but it is breaking on other pages. :(

Okay, No problem. If you would like to work on this issue, I will assist you if you have any doubt or something.

**Pull request closed without merging**